src/js/components/account/user-registration.js

```diff
@@ -124,6 +140,17 @@ export class UserRegistration {
 
     $form.submit((event) => {
       event.preventDefault();
+
+      // Check if the user has read the privacy policy
+      const $childDiv = $privacyContainer.find(".w-checkbox-input")
+      // (our custom component has this class when it is checked)
+      const $isRead = $childDiv.hasClass("w--redirected-checked");
```

Comment:
I do not think this actually returns a jQuery object so, perhaps just:

```suggestion
      const isRead = $childDiv.hasClass("w--redirected-checked");
```